lms/static/sass/multicourse/_course_about.scss

```diff
@@ -164,28 +164,6 @@
             }
           }
 
-          span.register, span.add-to-cart, span.sneakpeek {
```

Comment:
Is `add-to-cart` old or new shopping cart?

How does this file `diff` against upstream now?